init.lua

```diff
@@ -420,9 +420,9 @@ require('lazy').setup({
       -- NOTE: `opts = {}` is the same as calling `require('fidget').setup({})`
       { 'j-hui/fidget.nvim', opts = {} },
 
-      -- `neodev` configures Lua LSP for your Neovim config, runtime and plugins
+      -- `lazydev` configures Lua LSP for your Neovim config, runtime and plugins
       -- used for completion, annotations and signatures of Neovim apis
-      { 'folke/neodev.nvim', opts = {} },
+      { 'folke/lazydev.nvim', ft = 'lua', opts = {} },
```

Comment:
To avoid an LSP error on https://github.com/nvim-lua/kickstart.nvim/blob/20fa60a709e30fcf9ab6060e20750841141f7937/init.lua#L207

perhaps this should use the configuration suggested in https://github.com/folke/lazydev.nvim/tree/6184ebbbc8045d70077659b7d30c705a588dc62f#-installation
```suggestion
      {
        'folke/lazydev.nvim',
        ft = 'lua',
        opts = {
          library = {
            -- Load luvit types when the `vim.uv` word is found
            { path = 'luvit-meta/library', words = { 'vim%.uv' } },
          },
        },
      },
      { 'Bilal2453/luvit-meta', lazy = true },
```

Reply:
I got the sense that these are optional configs for those who want specific vim.uv LSP typings. Not everyone will need/want them. However, if there's no downside and or most people need them, I see no problem with it.

At the risk of bloat perhaps someone else can chime in?

Reply:
I feel like, for a kickstart config, the suggested would be the preferred aproach. Anyone who doens't want this will propably be more awere of how to remove it.

Reply:
As it stands, on `master` using `folke/neodev.nvim` we don't see an "undefined field `fs_stat`" LSP error. But with the change in this PR, we do see that error (unless we apply the suggestion above). That's why I think we might want to do follow the configuration suggested in https://github.com/folke/lazydev.nvim/tree/6184ebbbc8045d70077659b7d30c705a588dc62f#-installation - Otherwise, merging this PR as-is will leave the fresh `kickstart.nvim` installation with an LSP error in `init.lua`, which IMO degrades the desired "first timer" experience.